### DISCUSSION TITLE
convert this image into black and white but getting some issue

### DISCUSSION THREAD

**Discussion by AshaRxKhan** [dlemstra/Magick.NET#1251] [Help]
Hello 
I am trying to convert this image magick command into c# code.
D:/input.png +dither -colors 256 -normalize -flatten -monochrome +compress D:/output.png
**input file**
![a70acace-d4df-461c-a6c9-a2bea1bdbda6](https://user-images.githubusercontent.com/113311036/189546912-413c3cf8-02b0-40ca-98ae-3d08bf6f2adf.png)
**expected output**
![640602](https://user-images.githubusercontent.com/113311036/189546928-08e796fd-a8c9-4415-84af-0e17e2c0af45.png)
-----------------------
 QuantizeSettings settings = new QuantizeSettings();

            settings.DitherMethod = DitherMethod.No;
             settings.Colors = 256;
            
             MagickImageCollection magickImageCollection = new MagickImageCollection();
             MagickReadSettings magickReadSettings = new MagickReadSettings();
            //magickReadSettings.UseMonochrome = false;
             MagickImage image = new MagickImage(intput, magickReadSettings);
             
             image.Quantize(settings);

             image.Normalize();
             image.Write(output);
             magickImageCollection.Read((output));
             image = (MagickImage)magickImageCollection.Flatten();
            image.ColorType = ColorType.Bilevel;
           image.Settings.Compression=CompressionMethod.NoCompression;
         
             image.Write(output");

        }
        **output after c# conversion**
![B44 NoCompressionwitFloydSteinbergr](https://user-images.githubusercontent.com/113311036/189547005-23135861-eadc-423d-a041-4b74281ecc69.png)

**Comment by dlemstra** [comment]
Sorry for the late response but are you using the same version of ImageMagick as Magick.NET?
